README.md

```diff
@@ -13,27 +13,18 @@ A program for organizing Endless Sky installations.
 ESLauncher2 requires DirectX 11 or 12, Vulkan or Metal. See also [#4](https://github.com/EndlessSkyCommunity/ESLauncher2/issues/4).
 
 
-### Installation instructions for Mac
-#### Additional requirements for Mac ####
-**Important**: currently, the launcher works only when run from a console which has access to the full filesystem.
-We are working on a solution to overcome this.
+### Additional instructions for Mac
+**Important**: the security mechanisms of newer MacOS system do not allow to run
+unsigned applications. Therefore, please follow the steps below the first time
+you run the launcher on MacOS.
 
 #### Instructions ####
 - Download the zip to a location of your choice
-- Open a Terminal session and navigate to the zip file
-- Unzip the launcher (`unzip eslauncher2-x86_64-apple-darwin.zip`)
-- mark it as executable (`chmod 755 eslauncher2-x86_64-apple-darwin`)
-- run the launcher (`./eslauncher2-x86_64-apple-darwin`)
-
-#### Full Disk Access for the Console ####
-- Open *System Preferences*
-- Navigate to *Security & Privacy*
-- Select the *Privacy* tab
-- Scroll down to the item *Full Disk Access*
-- If *Terminal* is not present or not ticked, add it or activate the tick box
-- To add Terminal:
-  * Unlock the settings by clicking on the lock symbol
-  * If *Terminal* is not there, click on the plus
-  * Navigate to *Applications* -> *Utilities* -> *Terminal*
-  * Select it and click *Open*
-  * It should now appear in the list, make sure the box before it is ticked
+- Open Finder and navigate to that location
+- Unzip the file (usually double clicking on it is enough)
+- Right click on the extracted ESLauncher2.app and select "Open"
+- If you get presented with a dialogue with 2 options, click cancel and repeat
+above step
+- Once you see the dialogue with 3 options, select "Open"
```

Comment:
Can we make this less vague, perhaps include screenshots?

Reply:
Good point, I had already thought about that, but didn't do it because I couldn't attach screen shots easily. But I found a way.